mne/epochs.py

```diff
@@ -1935,7 +1935,7 @@ class EpochsArray(BaseEpochs):
         If None (default), all event values are set to 1 and event time-samples
         are set to range(n_epochs).
     tmin : float
-        Start time before event. If nothing provided, defaults to -0.2.
+        Start time before event. If nothing provided, defaults to 0.
```

Comment:
this is changing the default behavior for the `EpochsArray`. is this what we want?

Reply:
This is fixed in `master`, not sure why it shows up here -- probably some history problem